### DISCUSSION TITLE
Hit error icons confusing

### DISCUSSION THREAD

**Discussion by jxu** [ppy/osu#18128] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/7989982/167226916-b4237e34-2578-4b95-b902-2fdae61f43cd.png)

What are the icons? Is it a truck and bicycle? Does truck mean too soon and bicycle mean too late? I think it's clearer if the icons were removed and we had an abstract timing bar (or have the words early and late, or rabbit and turtle) it is confusing to me why these icons were chosen. Idk about you but bicycles in my area can travel pretty quickly, as fast or faster than surrounding vehicle traffic, while trucks have to move slower due to being heavier.

**Comment by integerrr** [comment, marked as answer]
#11915, see peppy's comment about the icon being a placeholder design

In addition, the icons can now be removed through the skin editor or changed to text instead 
<img width="628" alt="image" src="https://user-images.githubusercontent.com/7279624/167229220-4d0ae64f-8df1-440d-aa21-29888e495ed4.png">
